parsers/ocaml.c

```diff
@@ -873,23 +920,33 @@ static void killCurrentState ( void )
 		/* nothing more */
 		break;
 	}
+	return popped;
 }
 
-/* used to prepare tag for OCaml, just in case their is a need to
+/* Keep track of our _true_ line number and file pos,
+ * as the lookahead token gives us false values. */
+static unsigned long ocaLineNumber;
+static fpos_t ocaFilePosition;
+
+/* used to prepare an OCaml tag , just in case their is a need to
```

Comment:
while at it changing this comment, let's fix it: s/their/there/, remove space before the comma